.circleci/config.yml

```diff
@@ -274,8 +275,13 @@ jobs:
             coverage report || true
             coverage html || true
             coveralls || true
+      - run:
+          name: Metric aggregation
+          command: |
+            source .venv/bin/activate
+            python scripts/metric_aggregator.py . amd64
       - store_artifacts:
-          path: htmlcov/
+          path: metrics/
```

Comment:
could we maybe concretize the naming a bit here? for someone who's reading it the first time i think "metrics" can pretty much mean anything. could we say things like `Parity metric aggregation` amd `path: partiy_metrics` etc?

Reply:
I renamed the target-name here, and the output-dir of the aggregated parity metrics to `parity_metrics`.
As discussed, we keep the names for other code-parts for now (pytest plugin, handler, etc).